examples/servant-elm-street-main/src/ExampleAPI.hs

```diff
@@ -9,25 +9,28 @@
 {-# LANGUAGE TypeOperators #-}
 
 module ExampleAPI
-  ( queryFile,
+  ( userAPI,
+    LangElm,
     server,
   )
 where
 
 import Data.Aeson (ToJSON)
-import Data.Text (Text)
 import Elm.Ast (ElmDefinition)
 import Elm.Generic (Elm (..))
-import Elm.Print (showDoc)
 import GHC.Generics (Generic)
-import Prettyprinter (vsep)
-import Servant (Get, JSON, Proxy (..), Server, type (:<|>) (..), type (:>))
+import Servant (Get, JSON, Post, Proxy (..), ReqBody, Server, type (:<|>) (..), type (:>))
 import Servant.Foreign (HasForeignType (..))
-import ServantElm (elmForAPI)
+
+data LangElm
+
+instance Elm a => HasForeignType LangElm ElmDefinition a where
+  typeFor _ _ proxyA = toElmDefinition proxyA
```

Comment:
I don't think this should be here - it's already present in the library.

Reply:
done